command/task_delete_test.go

```diff
@@ -65,7 +66,7 @@ func TestTaskDeleteCommand_AutocompleteArgs(t *testing.T) {
 			}
 
 			tasksResponse := oapigen.TasksResponse{
-				RequestId: "!@#$%^&*()?!abc",
+				RequestId: uuid.New(),
```

Comment:
Not sure what the old string was, I assume it was supposed to indicate a randomized string